docs/introduction_tutorial/intro_tutorial_part2.ipynb

```diff
@@ -0,0 +1,1228 @@
+{
```

Comment:
<p>I would like to work on improving our repr (see #299). No action needed here, but we will want to regenerate this once that is done.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pangeo-forge/pangeo-forge-recipes/pull/293/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>